src/App.tsx

```diff
@@ -1,25 +1,46 @@
-import React from 'react';
-import './App.css';
+import React, { useState } from 'react';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
+import './App.css';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [perPage, setPerPage] = useState(5);
+
+  const handlePageChange = (page: number) => {
+    setCurrentPage(page);
+  };
+
+  const handlePerPageChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
```

Comment:
When changing the items per page, the current page is reset to 1. This is a good practice to ensure that the user is taken back to the start of the list to avoid confusion.